addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/internal/AuthenticationBrowserHook.java

```diff
@@ -72,14 +72,16 @@ private ZestBasicRunner getZestRunner(WebDriver webDriver) {
 
     @Override
     public void browserLaunched(SeleniumScriptUtils ssUtils) {
-        ZestBasicRunner runner = getZestRunner(ssUtils.getWebDriver());
+        ZestAuthRunner runner = getZestRunner(ssUtils.getWebDriver());
```

Comment:
Is it safe to use the same runner instance for multiple browser launches (e.g. is it possible for two browsers be used in parallel with different users / script params?)

Reply:
It's definitely possible but the current expectation/implementation is that there will be no user auth concurrency (see `AuthUtils.enableBrowserAuthentication`).